lib/metadata/in_memory/backend.js

```diff
@@ -41,7 +42,8 @@ const metastore = {
             if (!metadata.buckets.has(bucketName)) {
                 return cb(errors.NoSuchBucket);
             }
-            return cb(null, metadata.buckets.get(bucketName));
+            return cb(null, BucketInfo.fromObj(
```

Comment:
why are we returning an object for the bucket if we call getBucketAttributes but a string if we are returning an object and a bucket?  I believe this had to do with some double stringifying in metadata which should be fixed.